userguide/storage.rst

```diff
@@ -1788,10 +1788,10 @@ Importing a Disk
 
 The :menuselection:`Pool --> Import Disk` screen, shown in
 :numref:`Figure %s <zfs_import_disk_fig>`, is used to import a
-**single** disk that is formatted with a UFS, NTFS, MSDOS, or EXT2
-filesystem. The import is meant to be a temporary measure to copy the
-data from a disk to an existing ZFS dataset. Only one disk can be
-imported at a time.
+**single** disk that is formatted with UFS (BSD Unix), FAT or
+NTFS (Windows), or EXT2 (Linux) filesystems. The
+import is meant to be a temporary measure to copy the data from a disk
+to an existing ZFS dataset. Only one disk can be imported at a time.
```

Comment:
Again, I'm not sure if the "one disk" thing is truly valid.

Reply:
Confirmed that it is one at a time by Mav and Caleb.